packages/@aws-cdk/aws-opensearchservice/package.json

```diff
@@ -115,6 +130,7 @@
     "@aws-cdk/aws-secretsmanager": "0.0.0",
     "@aws-cdk/core": "0.0.0",
     "@aws-cdk/custom-resources": "0.0.0",
+    "@aws-cdk/integ-tests": "0.0.0",
```

Comment:
But not needed here.

Reply:
Is there a guideline of how internal dependencies should be added? From what I can see, it seems to be standard to add them as peerDependencies, but if not added as devDependencies as well then the build produces warnings, which doesn't seem right?

Reply:
hmm. I'm not actually sure if we have this documented anywhere but basically, if it doesn't produce an error, don't add it.

Reply:
Alright, will change this back

Reply:
After taking a look, I think the warnings may actually be the result of these being declared as both peer dependencies *and* normal dependencies. But I'll check that theory in another PR with the documentation updates.

Reply:
Keep in mind some of why we do these oddities is that we translate these libraries into a half dozen languages and between v1 and v2 structure/conventions. Some of it might not align with what you've seen elsewhere because of that.